app/models/invite.rb

```diff
@@ -160,6 +183,44 @@ def self.create_invite_by_email(email, invited_by, opts = nil)
     invite
   end
 
+  # multiple use invite link
+  def self.generate_multiple_use_invite_link(invited_by, opts = nil)
```

Comment:
again the question is... is this `generate_invite_link`?

also ... lets avoid ... opts = nil in favour of being explicit: `invited_by, group_ids: nil, max_redemptions_allowed: 1` and so on.

Reply:
The two methods `generate_single_use_invite_link` and `generate_multiple_use_invite_link` are doing very different things. `generate_single_use_invite_link` is falling back to `create_invite_by_email` method for doing all the heavy lifting while `generate_multiple_use_invite_link` is a standalone method and do not share any code with `create_invite_by_email`.  So combining these two methods will not be trivial.

Agreed with `opts = nil`. I'll replace this with keyword arguments.

Reply:
Updated `generate_multiple_use_invite_link` method to use keyword arguments.